node/libs/roles/src/node/messages.rs

```diff
@@ -14,10 +14,13 @@ pub enum Msg {
     SessionId(SessionId),
 }
 
-impl Msg {
-    /// Get the hash of this message.
-    pub fn hash(&self) -> MsgHash {
-        MsgHash(keccak256::Keccak256::new(&zksync_protobuf::canonical(self)))
+impl ByteFmt for Msg {
+    fn decode(bytes: &[u8]) -> anyhow::Result<Self> {
+        zksync_protobuf::decode(bytes)
+    }
+
```

Comment:
don't expose the bytefmt of the message. Keep it an implementation detail of signing.

Reply:
also don't define the decode